components/Navigation/index.js

```diff
@@ -0,0 +1,139 @@
+import { useState } from "react";
+import Link from "next/link";
+import styled from "styled-components";
+
```

Comment:
💡 **A general advise**
_Choosing meaningful and consistent naming for your styled components can greatly enhance the readability and maintainability of your code._

For example: 

```js

const MenuItem = styled.li`
  /* Styles for top-level navigation items */
`;

const SidebarMenuItem = styled(MenuItem)`
  /* Styles for sidebar menu items */
`;

const MenuList = styled.ul`
  /* Styles */
`;

const NavLink = styled.a`
  /* Styles */
`;


```